### ISSUE TITLE
README.md link http://imagej.net/Kotlin_Scripting 404

### ISSUE THREAD

**Issue by jayvdb** [scijava/scripting-kotlin#11]
http://imagej.net/Kotlin_Scripting is 404

It seems that page doesnt exist . c.f. https://github.com/search?q=repo%3Aimagej%2Fimagej.github.io%20kotlin&type=code and https://github.com/imagej/imagej.github.io/tree/main/_pages/scripting

**Comment by ctrueden** [comment]
Yes, the kotlin scripting support in SciJava is immature, and no page was ever made on the ImageJ wiki to document it. Sorry about that. (FWIW, the page would now live at https://imagej.net/scripting/kotlin, were it to exist... but it does not.)
